installer.sh.in

```diff
@@ -1031,9 +1081,10 @@ failed to activate swap on $dev!\ncheck $LOG for errors." ${MSGBOXSIZE}
             ext2) MKFS="mke2fs -F"; modprobe ext2 >$LOG 2>&1;;
             ext3) MKFS="mke2fs -F -j"; modprobe ext3 >$LOG 2>&1;;
             ext4) MKFS="mke2fs -F -t ext4"; modprobe ext4 >$LOG 2>&1;;
-            f2fs) MKFS="mkfs.f2fs"; modprobe f2fs >$LOG 2>&1;;
+            f2fs) MKFS="mkfs.f2fs -f"; modprobe f2fs >$LOG 2>&1;;
```

Comment:
Why this change?

Reply:
I also explained the reason for this in the initial commit message. The reason is that if you don't do that, if you ask the installer to create a new f2fs filesystem on an f2fs partition, then it won't work (because by default mkfs.f2fs won't overwrite an existing f2fs filesystem, like the mkfs for other filesystems). It's the same reason why the "-F" option is present for the other mkfs commands